modules/auxiliary/scanner/http/wp_modern_events_calendar_sqli.rb

```diff
@@ -0,0 +1,99 @@
+##
```

Comment:
It looks like this file was accidentally included in this PR.

Reply:
oops, good catch